main/writer-ctags.c

```diff
@@ -38,10 +38,20 @@ struct rejection {
 	bool rejectionInThisInput;
 };
 
+static void *beginUCtagsFile (tagWriter *writer CTAGS_ATTR_UNUSED, MIO * mio CTAGS_ATTR_UNUSED,
+							  void *clientData CTAGS_ATTR_UNUSED)
+{
+#ifdef WIN32
+	if (Option.useSlashAsFilenameSeparator == FILENAME_SEP_UNSET)
+		Option.useSlashAsFilenameSeparator = FILENAME_SEP_USE_SLASH;
```

Comment:
It seems that casting is needed to remove `const` or need to define `OPTION_WRITE` before including "options_p.h".

```suggestion
		((optionValues *)&Option)->useSlashAsFilenameSeparator = FILENAME_SEP_USE_SLASH;
```

Reply:
Thank you.
However, I will take bit non-efficient technique as the first trial.
I remember the following commit when I see code like `#define SOMETHING const`.

```
commit 585d60d5a4ea4c6f2e96c9cd65dae87fe52d25c2
Author: Masatake YAMATO <yamato@redhat.com>
Date:   Tue Apr 29 16:13:57 2014 +0900

    Remove const modifier from File global variable declared in read.h
    
    This is a temorary bug fix.
    
    const modifier is added to `File' global variable declared in read.h.
    The intent is `File' should be modifiable only from read.c.
    From the outside the variable can be read.
    
    However, it causes the bug if -O[23] option is used when compiling.
    I found it in fortran.c.
    
      read_File();
      read_c_func_modifying_File();
      read_File();
    
    Though a field in `File' is modified in read_c_func_modifying_File.
    However, here gcc treats File as constant. gcc assigns the field to a
    CPU register. So the 2nd read_File cannot access to the value of the
    field updated by read_c_func_modifying_File.
```

Fixing the above issue took so much time.